src/Symfony/Component/Mailer/Bridge/Mailomat/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/mailomat-mailer",
+    "type": "symfony-mailer-bridge",
+    "description": "Symfony Mailomat Mailer Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Patrick Landolt",
+            "email": "patrick.landolt@artack.ch"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/mailer": "^7.2"
+    },
+    "require-dev": {
+        "symfony/http-client": "^7.2",
+        "symfony/http-foundation": "^7.1",
+        "symfony/webhook": "^7.2"
+    },
+    "conflict": {
```

Comment:
There should be entries for `symfony/http-client` and `symfony/webhook` too.

Reply:
We should forbid versions that we do not use when running tests (i.e. `"symfony/http-client": "<7.2"` and `"symfony/webhook": "<7.2"`.

Reply:
By the way, do we really need that recent versions? 6.4 should work too, right?

Reply:
Yes, I alligned the versions to `^6.4|^7.0` as most of the other Bridges also use.

Reply:
We don't have those conflict rules in other bridges so LGTM as is (keeping only HttpFoundation one and widening the version range)